.eslintrc.js

```diff
@@ -0,0 +1,128 @@
+module.exports = {
+  root: true,
+
+  // Make ESLint compatible with TypeScript
```

Comment:
Why are `.eslintrc.js` and `.eslintignore` popping up. This is probably an issue with git, maybe some issues with an old merge.